example/lambdas/ftpPopulateTestLambda/index.js

```diff
@@ -131,10 +131,8 @@ const deleteFtpGranules = async ({ filePaths }) => {
 
 const handler = async (event, _context) => {
   log.info(event);
-  if (event.command === 'delete') {
-    return await deleteFtpGranules(event);
-  }
-  return await uploadFtpGranuleDataForDiscovery(event);
+  if (event.command === 'delete') return deleteFtpGranules(event);
+  return uploadFtpGranuleDataForDiscovery(event);
```

Comment:
I'm not sure if we want this change 🤔. We generally use `return await` in try/catch blocks to preserve error handling (https://jakearchibald.com/2017/await-vs-return-vs-return-await/). This doesn't appear to be the case here so it may be redundant but if it's not a functional improvement or fix I'm not sure if there's value in changing it.

Reply:
It makes sense Nate.
I was just removing what was redundant but I didn't really think about the whole context.
Thanks for the feedback and the review!